less/switchControls.less

```diff
@@ -85,3 +85,6 @@
     }
   }
 }
+.pullRight {
```

Comment:
++ please feel free to push to this branch (but do not rebase or squash commits)